mempool/src/feerate.rs

```diff
@@ -51,11 +62,12 @@ impl FeeRate {
         self.atoms_per_kb
     }
 
-    // TODO Use NonZeroUsize for divisor
-    fn div_up(dividend: u128, divisor: usize) -> u128 {
-        debug_assert!(divisor != 0);
-        let divisor = u128::try_from(divisor).expect("div_up conversion");
-        (dividend + divisor - 1) / divisor
+    fn div_up(dividend: Amount, divisor: NonZeroUsize) -> Amount {
```

Comment:
Please call this `div_and_round_up` or something that indicates that it's dividing then rounding up. `div_up` is not an agreed upon naming standard that I know of.

Reply:
Following your other comment, I changed our implementation to follow Bitcoin Core (there was no good reason why I diverged), and the need for this function disappeared.

Reply:
I also changed `FeeRate`'s underlying type from `u128` to `Amount`.